tasks/main-clevis.yml

```diff
@@ -11,6 +11,7 @@
   service:
     name: clevis-luks-askpass.path
     enabled: true
+    daemon_reload: true
```

Comment:
Does `daemon_reload` work with the `service` module? I thought it would work only with the `systemd` module (in which case perhaps we should move to it?)

Reply:
> Does `daemon_reload` work with the `service` module?

Yes.  The `service` module is basically just a wrapper around the platform specific module.  Any parameters not understood by the service module are passed through to the underlying platform specific module, in this case, systemd.

> I thought it would work only with the `systemd` module (in which case perhaps we should move to it?)

If you want to make it explicit, then yes, we should move to the `systemd` module.